Add comment to explain rnn bias vectors

ajaech opened the pull request:
Summary: RNN/LSTMs only need one bias vector, but our implementation uses two to be compatible with CuDNN. This diff adds a comment to explain this.

Differential Revision: D13602365

zou3519 commented on torch/nn/modules/rnn.py:
```diff
@@ -69,6 +69,7 @@ def __init__(self, mode, input_size, hidden_size,
                 w_ih = Parameter(torch.Tensor(gate_size, layer_input_size))
                 w_hh = Parameter(torch.Tensor(gate_size, hidden_size))
                 b_ih = Parameter(torch.Tensor(gate_size))
+                # Second bias vector included for CuDNN compatibility
```
Nit: maybe add a little more information "standard definition only uses one bias vector"

zou3519 reviewed (approved):
go for it

ajaech committed:
Add comment to explain rnn bias vectors (#15843)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/15843

RNN/LSTMs only need one bias vector, but our implementation uses two to be compatible with CuDNN. This diff adds a comment to explain this.

Reviewed By: ezyang

Differential Revision: D13602365

fbshipit-source-id: c2293365cffe1de4e0da4874cfff916ef5d05054
